Makefile

```diff
@@ -50,28 +50,34 @@ endif
 HOSTS=127.0.0.1 world.productopener.localhost fr.productopener.localhost static.productopener.localhost ssl-api.productopener.localhost fr-en.productopener.localhost
 # commands aliases
 DOCKER_COMPOSE=docker compose --env-file=${ENV_FILE} ${LOAD_EXTRA_ENV_FILE}
-# docker command that do not need the shared network
-DOCKER_COMPOSE_BUILD=COMPOSE_FILE="${COMPOSE_FILE_BUILD}" ${DOCKER_COMPOSE}
+# Ensure shared_network is referenced when running locally
+DOCKER_COMPOSE_RUN=COMPOSE_FILE="${COMPOSE_FILE};docker/run.yml" ${DOCKER_COMPOSE}
```

Comment:
@hangy I think there might have been a merge problem.
I think we did this on the main branch and revert it after.
So I don't think you need this change, as well as other changes that are underneath.